src/deroff.rs

```diff
@@ -296,6 +326,152 @@ impl Deroffer {
             c => c.chars().all(|c| c.is_whitespace()),
         }
     }
+
+    // Replaces the g_macro_dict lookup in the Python code
+    fn g_macro_dispatch<'a>(&mut self, s: &'a str) -> bool {
```

Comment:
Lifetime annotation isn't required here.
```suggestion
    fn g_macro_dispatch(&mut self, s: &str) -> bool {
```